addon/templates/components/bs-form.hbs

```diff
@@ -4,7 +4,7 @@
   novalidate={{this.hasValidator}}
   class={{this.layoutClass}}
   ...attributes
-  {{ref this "_element"}}
+  {{create-ref "mainNode"}}
```

Comment:
There is a conflict here now, after #1183 has been merged, that removes ref-usage all together. @lifeart could you please rebase and remove it from here (also from the backing component).